annotation-web/src/main/java/eu/europeana/annotation/web/model/BatchProcessingStatus.java

```diff
@@ -1,8 +1,13 @@
 package eu.europeana.annotation.web.model;
 
 import java.util.HashMap;
+import java.util.Map;
 
 public class BatchProcessingStatus implements BatchReportable {
+	public static final String successCountStr="Success count: ";

```

Comment:
these constants are not used in this class, but only in tests, need to be moved there